RaceTrack/Tournament/Tournament.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DataRace;
+
+namespace RaceTournament
+{
+    public class TournamentResult
+    {
+        private Queue<Racer> _racers;
+        public Queue<Racer> _winners = new Queue<Racer>();
```

Comment:
It is better in my opinion to work with concurrent collections here, because you are working with multithreading